includes/functions.php

```diff
@@ -1541,24 +1541,41 @@ function pmpro_updateMembershipCategories( $level, $categories ) {
 /**
  * pmpro_getMembershipCategories() returns the categories for a given level
  *
- * @param int $level_id is a valid membership level ID
- *
- * @return int[]
+ * @param int $level_id The membership level ID.
+ * @return array List of category IDs.
  */
-function pmpro_getMembershipCategories( $level_id ) {
+function pmpro_get_membership_categories( $level_id ) {
```

Comment:
```suggestion
function pmpro_getMembershipCategories( $level_id ) {
```
Changing the function name will break all references to this function

Reply:
My bad— I must've ran my "camel case the things" filter in VS Code. Fixed.